sdk/src/main/java/com/mapbox/maps/MapControllable.kt

```diff
@@ -73,4 +74,10 @@ interface MapControllable : MapboxLifecycleObserver {
    * Set [OnFpsChangedListener] to get map rendering FPS.
    */
   fun setOnFpsChangedListener(listener: OnFpsChangedListener)
+
+  /**
+   * Add static image widget to the map.
+   */
+  @MapboxExperimental
```

Comment:
not sure we need experimental in internal classes, mostly it's indicating that this API may be broken

Reply:
But that's public interface right

Reply:
imo it's enough to declare experimental to interface and not duplicate it in a class that implements it. afair lint works fine
in your case we're doubling that annotation

Reply:
I got it, this is an interface, sorry, messed up with another class 😢